v2/datastream-to-mongodb/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToMongoDB.java

```diff
@@ -0,0 +1,237 @@
+ * they are automatically added to MongoDB.
+ *
+ * <p>Example Usage: TODO: FIX EXMAPLE USAGE
+ *
+ */
+public class DataStreamToMongoDB {
+
+  private static final Logger LOG = LoggerFactory.getLogger(DataStreamToMongoDB.class);
+  private static final String AVRO_SUFFIX = "avro";
+  private static final String JSON_SUFFIX = "json";
+  public static final Set<String> MAPPER_IGNORE_FIELDS = new HashSet<String>(
+          Arrays.asList(
+                  "_metadata_stream", "_metadata_schema", "_metadata_table", "_metadata_source",
+                  "_metadata_ssn", "_metadata_rs_id", "_metadata_tx_id",
+                  "_metadata_dlq_reconsumed", "_metadata_error", "_metadata_retry_count",
+                  "_metadata_timestamp", "_metadata_read_timestamp", "_metadata_read_method",
+                  "_metadata_source_type", "_metadata_deleted", "_metadata_change_type",
+                  "_metadata_log_file", "_metadata_log_position"));
+
+  /** The tag for the main output of the json transformation. */
+  public static final TupleTag<FailsafeElement<String, String>> TRANSFORM_OUT =
+          new TupleTag<FailsafeElement<String, String>>() {};
+
+
+  /**
+   * Options supported by the pipeline.
+   *
+   * <p>Inherits standard configuration options.</p>
+   */
+  public interface Options extends PipelineOptions, StreamingOptions {
+    @Description("The GCS location of the avro files you'd like to process")
+    String getInputFilePattern();
+    void setInputFilePattern(String value);
+
+    @Description("The GCS output format avro/json")
+    @Default.String("json")
+    String getInputFileFormat();
+    void setInputFileFormat(String value);
+
+    @Description(
+        "The Pub/Sub subscription with DataStream file notifications."
+            + "The name should be in the format of "
+            + "projects/<project-id>/subscriptions/<subscription-name>.")
+    String getInputSubscription();
+    void setInputSubscription(String value);
+
+    @Description("The DataStream Stream to Reference.")
+    String getStreamName();
+    void setStreamName(String value);
+
+    @Description("The starting DateTime used to fetch from GCS (https://tools.ietf.org/html/rfc3339).")
+    @Default.String("1970-01-01T00:00:00.00Z")
+    String getRfcStartDateTime();
+    void setRfcStartDateTime(String value);
+
+    @Description("The number of concurrent DataStream files to read.")
+    @Default.Integer(30)
+    Integer getFileReadConcurrency();
+    void setFileReadConcurrency(Integer value);
+
+    // Destination MongoDB Connection String
+    @Description("The MongoDB Atlas connection string")
+    @Default.String("{_metadata_dataset}")
+    String getMongoDBUri();
+    void setMongoDBUri(String value);
+
+    @Description("The MongoDB Database name")
+    @Default.String("{_metadata_dataset}")
+    String getDatabase();
+    void setDatabase(String value);
+
+    @Description("The MongoDB Collection")
+    @Default.String("{_metadata_dataset}")
```

Comment:
Improve default or remove if there should not be a default?

Reply:
Removed defaults